CONTRIBUTING.md

```diff
@@ -29,6 +29,25 @@ This is a rough outline of what a contributor's workflow looks like:
 
 Thanks for contributing!
 
+### Local setup
+
+```sh
+make make deploy-full-local-setup
```

Comment:
make duplicated

Reply:
thanks for the catch, amended

Reply:
basically lgtm, but do we need `deploy-full-local-setup` anymore ?